llvm/lib/Passes/PassBuilderBindings.cpp

```diff
@@ -28,11 +29,14 @@ class LLVMPassBuilderOptions {
 public:
   explicit LLVMPassBuilderOptions(
       bool DebugLogging = false, bool VerifyEach = false,
+      const char *AAPipeline = nullptr,
```

Comment:
`LLVMPassBuilderOptions` is never constructed with params. the constructor shouldn't take any arguments.

I'll do this in a followup patch